bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap_de.properties

```diff
@@ -4,11 +4,11 @@ binding.openweathermap.description = OpenWeatherMap - Aktuelles Wetter und Progn
 
 # bridge types
 thing-type.openweathermap.weather-api.label = OpenWeatherMap Konto
-thing-type.openweathermap.weather-api.description = Erm�glicht den Zugriff auf die OpenWeatherMap API.
+thing-type.openweathermap.weather-api.description = Erm�glicht den Zugriff auf die OpenWeatherMap API.
```

Comment:
Can you check the file encoding? It needs to be ISO-8859.

Reply:
corrected encoding (hopefully :-) )